### PR TITLE
Fallback for `object`

### LINKED ISSUE #27300: Examples should include fallback content
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/object

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

Examples should include [fallback content](https://html.spec.whatwg.org/multipage/dom.html#fallback-content)

### What did you expect to see?

Examples should include [fallback content](https://html.spec.whatwg.org/multipage/dom.html#fallback-content)

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/object`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/object
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/object/index.md
* Last commit: https://github.com/mdn/content/commit/62d47afb8e8affdbf65374e1b7a55589831b8cea
* Document last modified: 2023-05-19T21:35:57.000Z

</details>

### PR DESCRIPTION
**Pull request by postmeback**
Added example for fallback content

"Fixes #27300"

### CODE REVIEW

**Comment by postmeback** [comment]
Requesting any maintainer to help me with `prettier` configuration.

File: files/en-us/web/html/element/object/index.md [hunk 1]
```diff
@@ -55,13 +55,28 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
   type="video/mp4"
   data="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm"
   width="600"
-  height="140"></object>
+  height="140"
+></object>
```
Comment (estelle) [inline files/en-us/web/html/element/object/index.md:59]:
```suggestion
<img src="path/image.jpg" alt="useful image description" />
</object>
```

File: files/en-us/web/html/element/object/index.md [hunk 2]
```diff
@@ -55,13 +55,28 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
...
+```html
+<img
+  src="image.jpg"
+  alt="Example Image"
+  onerror="this.src='fallback.jpg'; this.alt='Fallback Image';"
+/>
+```
+
+In this example, the <img> tag is used to display an image with the src attribute pointing to "image.jpg". However, if the image fails to load, the onerror event handler is triggered. Inside the event handler, we set the src attribute to "fallback.jpg" and the alt attribute to "Fallback Image". This ensures that if the original image cannot be loaded, the fallback image will be displayed instead, providing a fallback content option for users.
+
```
Comment (estelle) [inline files/en-us/web/html/element/object/index.md:79]:
```suggestion
If the video in the example fails to load, the user will be provided with an image as fallback content. The {{HTMLElement("img")}} tag is used to display an image. We include the `src` attribute set to the path to the image we want to embed. We also include the `alt` attribute, which provides the image with an accessible name. If the image also fails to load, the content of the `alt` attribute will be displayed.

```

File: files/en-us/web/html/element/object/index.md [hunk 3]
```diff
@@ -55,13 +55,28 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
   type="video/mp4"
   data="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm"
   width="600"
-  height="140"></object>
+  height="140"
```
Comment (estelle) [inline files/en-us/web/html/element/object/index.md:58]:
```suggestion
  height="140">
```
this issue was caused by running prettier :|

File: files/en-us/web/html/element/object/index.md (hunk 2 above)
Comment (estelle) [reply files/en-us/web/html/element/object/index.md:79]: Thank you for your PR. The fallback content should be within the `object` example, not as a separate fallback section. The issue raised was about providing a fallback for when the resource defined by the `object` attribute fail to load, not if the image fails to load ;)

We try not to include "onEvent" on MDN, opting for addEventListener instead, just to demonstrate best practices.

**Review by estelle — CHANGES_REQUESTED** [review]

File: files/en-us/web/html/element/object/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/html/element/object/index.md:59]: I think https://github.com/mdn/content/issues/27300 was about adding fallback content for the object, not an example of fallback content in general. So, suggested changes

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**